gpytorch/settings.py

```diff
@@ -661,3 +662,16 @@ class use_toeplitz(_feature_flag):
     """
 
     _default = True
+
+
+class verbose(_feature_flag):
+    """
+    Print out information whenever running an expesnive linear algebra routine (e.g. Cholesky, CG, Lanczos, CIQ, etc.)
+    """
+
+    _default = False
+
+    def __init__(self, state=True):
+        super().__init__(state=state)
+        if state:
+            logging.basicConfig(level=logging.DEBUG)
```

Comment:
Hmm looks like with this logic the logging level will always be DEBUG and never reset? Should this instead overwrite the `__enter__` and `__exit__` methods?

Reply:
Yeah that's probably a good call.